applications/ipc_radio/src/bt_hci_ipc.c

```diff
@@ -283,8 +283,10 @@ static void queue_thread(void)
 		}
 
 		err = ipc_service_release_rx_buffer(&hci_ept, (void *)block.ptr);
-		if (err) {
+		if (err < 0) {
 			LOG_ERR("Failed to release rx buffer: %d.", err);
+		} else if (err > 0) {
```

Comment:
```suggestion
		} else {
```
It may also be worth handling the case: err = 0. 0 is returned when success.

Reply:
Switched to "else"